app/v1/models/errors/ErrorWrapper.scala

```diff
@@ -29,6 +29,11 @@ case class ErrorWrapper(correlationId: String, error: MtdError, errors: Option[S
   def auditErrors: Seq[AuditError] =
     allErrors.map(error => AuditError(error.code))
 
```

Comment:
There was a class code-comment in individuals-income-received-api that it might be worth bringing over, i.e.
```
/** If the errors arg is defined, error should be a BadRequest.
  */
```
I think it's one of those things that's "you have to know it" otherwise.